src/SFA.DAS.ApprenticePortal/Models/ApprenticeHomepage.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace SFA.DAS.ApprenticePortal.Models
+{
+    public class ApprenticeHomepage
+    {
+        public Guid ApprenticeId { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string CourseName { get; set; }
+        public string EmployerName { get; set; }
+        public bool? ApprenticeshipComplete { get; set; }
```

Comment:
I think we will also need the Apprenticeship Stopped Status on the home page

Reply:
Added necessary columns required on the front end